src/App.tsx

```diff
@@ -1,14 +1,34 @@
 /* eslint-disable max-len */
-import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
+import { useEffect, useState } from 'react';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { TodoModal } from './components/TodoModal';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions } from './features/todos';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const [loading, setLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    getTodos()
+      .then(t => {
+        dispatch(actions.setTodos(t));
+      })
+      .catch(() => {
+        throw new Error('Failed to fetch todos');
```

Comment:
That's rude :)) 

Once again, in a fully set environment you will probably have something like a loadingTotosError in state, however, we have what we have so why don't we use `useState` once again to store a possible error and adjust our UI to display it? 
I mean, user should be aware if something went wrong.